pkg/dispatcher/config_test.go

```diff
@@ -105,8 +105,9 @@ var (
 	}
 
 	configWithBuildFarmWithJobs = Config{
-		Default: "api.ci",
-		KVM:     []api.Cluster{api.ClusterBuild02},
+		Default:  "api.ci",
```

Comment:
Looks like something we should clean up?

Reply:
In the test. Any non-empty string should be fine.